src/sankeyUtils/positionNode.ts

```diff
@@ -55,7 +55,7 @@ export function positionNode({
     },
     type: "end",
   })
-
+  if (nodeHeight < 0) nodeHeight = 5
```

Comment:
Think we should do `Math.max(nodeHeight, 5)` instead? Otherwise these nodes will appear larger than some nodes with higher flows